keras2onnx/_builtin.py

```diff
@@ -373,6 +373,36 @@ def _calc_explicit_padding(input_size, output_shape, output_padding, kernel_shap
     return pads
 
 
+@converter_func(TYPES.DepthToSpace)
+def convert_tf_depth_to_space(scope, operator, container):
+    if operator.target_opset < 11:
+        raise ValueError("DepthToSpace op is not supported for opset < 11")
```

Comment:
After adding `op_version` below in `add_node_with_output`, these two lines are not needed.

Reply:
Removed